docs/configuration/upstream/tests.md

```diff
@@ -70,6 +70,13 @@ If you want to trigger tests via pull request comment and not by every new commi
     manual_trigger: true
 ```
 
+:::info
+
+If you have both build and followup test jobs with `manual_trigger: true`, you will need to post both `/packit build` and
+after the builds are successful `/packit test` comment commands.
```

Comment:
I had to read it several times to understand it. I would suggest:
"...you will need to post both: first post `/packit build` command and
after the builds are successful post  `/packit test` comment command."

And nit-pick - s/followup/follow-up/ but some dictionaries recognize "followup" as a word.

Reply:
good points, updated